packages/test/test-service-load/src/runner.ts

```diff
@@ -463,6 +509,26 @@ async function setupOpsMetrics(container: IContainer, logger: ITelemetryLogger,
     };
 }
 
+function checkConfig(runConfig): Map<string, [number, number]> {
```

Comment:
rather than enforcing via string format, i'd rather we just store an object {min, max} directly in the config. this would then be usable for things that are not time as well

Reply:
rather than enforcing via string format, i'd rather we just store an object {min, max} directly in the config. this would then be usable for things that are not time as well